src/SwupFragmentPlugin.ts

```diff
@@ -58,22 +58,20 @@ export default class SwupFragmentPlugin extends PluginBase {
 
 	logger: Logger;
 
+	defaults: PluginOptions = {
```

Comment:
Why is this important to you?

Reply:
When overwriting or manipulating the plugin options from the outside, it's great to have access to the default or fallback version of them. Not sure why I give so much of a damn about this to be honest, haha. It's become almost philosophical at this point.